pkg/controller/node.go

```diff
@@ -21,7 +21,6 @@ import (
 	"fmt"
 
 	"github.com/go-logr/logr"
-	corev1 "k8s.io/api/core/v1"
 	v1 "k8s.io/api/core/v1"
```

Comment:
We should use `corev1` rather than `v1`. The former is more specific and there are many `v1` packages so including `core` is important to disambiguate among them.